Clojure compiler warning

When compiling JS which uses page.js it has the following annoying warning:

```
node_modules/page/page.js:1: WARNING - Suspicious code. The result of the 'not' operator is not being used.
!function(e){if("object"==typeof exports&&"undefined"!=typeof module)module.exports=e();else if("function"==typeof define&&define.amd)define([],e);else{var f;"undefined"!=typeof window?f=window:"undefined"!=typeof global?f=global:"undefined"!=typeof self&&(f=self),f.page=e()}}(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```
Would be cool to replace "!function(...){..}(...)" by "(function(...){...})(...)".

--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,14 @@
-BROWSERIFY=node_modules/.bin/browserify
+ROLLUP=node_modules/.bin/rollup
 INFOLOG := \033[34m ▸\033[0m
 
 page.js: index.js
 	@echo "$(INFOLOG) Building page.js.."
-	@$(BROWSERIFY) index.js --standalone page -o page.js
+	@$(ROLLUP) -c rollup.config.js
 
 watch:
 	find index.js | entr make page.js
 .PHONY: watch
+
+clean:
+	@rm page.js
+.PHONY: clean

--- a/index.js
+++ b/index.js
@@ -13,10 +13,10 @@
    */
 
   module.exports = page;
-  module.exports.default = page;
-  module.exports.Context = Context;
-  module.exports.Route = Route;
-  module.exports.sameOrigin = sameOrigin;
+  page.default = page;
+  page.Context = Context;
+  page.Route = Route;
+  page.sameOrigin = sameOrigin;
 
   /**
    * Short-cuts for global-object checks
@@ -25,6 +25,7 @@
   var hasDocument = ('undefined' !== typeof document);
   var hasWindow = ('undefined' !== typeof window);
   var hasHistory = ('undefined' !== typeof history);
+  var hasProcess = typeof process !== 'undefined';
 
   /**
    * Detect click event
@@ -601,8 +602,7 @@
   /**
    * Handle "click" events.
    */
-  /* jshint -W054 */
-  var hasProcess = new Function('return typeof process')() !== 'undefined';
+
   /* jshint +W054 */
   function onclick(e) {
     if (1 !== which(e)) return;

--- a/package.json
+++ b/package.json
@@ -17,14 +17,12 @@
     "test": "jshint index.js test/tests.js && mocha test/tests.js",
     "serve": "serve test",
     "test-cov": "jscoverage index.js index-cov.js; PAGE_COV=1 mocha test/tests.js -R html-cov > coverage.html",
-    "prepublish": "npm run make",
-    "make": "browserify index.js --standalone page -o page.js"
+    "make": "rollup -c rollup.config.js"
   },
   "dependencies": {
     "path-to-regexp": "~1.2.1"
   },
   "devDependencies": {
-    "browserify": "^6.3.2",
     "chai": "^1.10.0",
     "coveralls": "^2.11.2",
     "engine-dependencies": "^0.2.0",
@@ -35,6 +33,9 @@
     "jshint": "^2.5.10",
     "mocha": "^2.0.1",
     "mocha-lcov-reporter": "0.0.1",
+    "rollup": "^0.54.1",
+    "rollup-plugin-commonjs": "^8.2.6",
+    "rollup-plugin-node-resolve": "^3.0.2",
     "serve": "*",
     "should": "*"
   },

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,0 +1,20 @@
+import commonjs from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+
+export default {
+  input: 'index.js',
+  output: {
+    file: 'page.js',
+    format: 'umd',
+    name: 'page'
+  },
+  plugins: [
+    nodeResolve({
+      jsnext: true,
+      main: true
+    }),
+    commonjs({
+      include: ['node_modules/**', '**']
+    })
+  ]
+};

